vncviewer/EmulateMB.h

```diff
@@ -21,29 +21,50 @@
 
 #include <rfb/Timer.h>
 #include <rfb/Rect.h>
+#include <rdr/types.h>
 
 class EmulateMB : public rfb::Timer::Callback {
 public:
-  EmulateMB();
+  EmulateMB(rdr::U32 emulateMBModKey);
 
   void filterPointerEvent(const rfb::Point& pos, int buttonMask);
 
+  bool filterKeyPress(int keyCode, rdr::U32 keySym);
+  bool filterKeyRelease(int keyCode);
+
 protected:
   virtual void sendPointerEvent(const rfb::Point& pos, int buttonMask)=0;
 
   virtual bool handleTimeout(rfb::Timer *t);
 
+  virtual void writeKeyEvent(rdr::U32 keySym, rdr::U32 keyCode, bool down)=0;
```

Comment:
Again, let's try to have symmetry with the existing stuff. So I'd like a `sendKeyPress()` and `sendKeyRelease()` here.